imap_data_access/__init__.py

```diff
@@ -108,4 +108,4 @@
     "<start_date>(_<end_date>)_<version>.<extension>"
 )
 
-VALID_ANCILLARY_FILE_EXTENSION = {"cdf", "csv", "json"}
+VALID_ANCILLARY_FILE_EXTENSION = {"cdf", "csv", "json", "zip", "dat"}
```

Comment:
alphabetize the list of items.
```suggestion
VALID_ANCILLARY_FILE_EXTENSION = {"cdf", "csv", "dat", "json", "zip"}
```